packages/ui/src/utils/utils.ts

```diff
@@ -70,14 +70,36 @@ export const timeDistance = (date1 = 0, date2 = 0, onlyHighestDenomination = fal
  * @example
  * timeAgo(1708113838167) // Returns "1 hour ago"
  */
+
 export const timeAgo = (timestamp?: number | null): string => {
   if (timestamp === null || timestamp === undefined) {
     return 'Unknown time'
   }
 
+  const now = Date.now()
+  const threeDaysMs = 3 * 24 * 60 * 60 * 1000
```

Comment:
let's make this a optional second prop, which defaults to 3 days.
hardcoded is no-beuno.